test/integration/config/config-esm.test.js

```diff
@@ -7,16 +7,24 @@
 
 const test = require('node:test')
 const assert = require('node:assert')
+const semver = require('semver')
 
 const match = require('../../lib/custom-assertions/match')
 const util = require('util')
 const exec = util.promisify(require('child_process').exec)
 
 test('should gracefully handle ESM imports', async (t) => {
-  await t.test('when newrelic.js is misnamed', async () => {
```

Comment:
It's really just a different error when trying to load the cjs config under esm. If we want to account for the error and maintain the status quo, all we have to do is branch on process version. However, it is expected that this feature will be backported to 20.

Reply:
well that's not what this _should_ be testing. before it was validating that if you tried to require an esm config it failed. but now it works so i updated the config to be proper ESM to verify it will work

Reply:
let me tweak this even more